src/vmm_mad/remotes/lib/vcenter_driver/virtual_machine.rb

```diff
@@ -614,6 +601,7 @@ def get_vcenter_nics
                 else
                     network = device.backing.network
                 end
+                nic[:refs] = network.host.map {|h| h.parent._ref}
```

Comment:
This should have error checking in case h.parent or h.parent._ref does not exist